pkg/cmd/container/list.go

```diff
@@ -97,14 +97,10 @@ type ListItem struct {
 	Status    string
 	Runtime   string // nerdctl extension
 	Size      string
-	Labels    map[string]string
+	Labels    string
 	// TODO: "LocalVolumes", "Mounts", "Networks", "RunningFor", "State"
 }
 
-func (x *ListItem) Label(s string) string {
-	return x.Labels[s]
-}
-
```

Comment:
HI @AkihiroSuda 

The `Label` method is added by https://github.com/containerd/nerdctl/commit/2af4cef9e7e7ca2f36b3f81aa33f10b619e51e6b#diff-81c503cae22717c2cb5fb3e7c1dbee8d9d2a82d746f05d9b115cd251178e19dbR104 .

It not be used, so it may be removed :-)